inst/include/sampler.h

```diff
@@ -551,6 +548,21 @@ namespace smc {
         }
     }
     
+	template <class Space>
+	void sampler<Space>::Multinomial(unsigned n, unsigned k, arma::vec w, unsigned int * X) {		
```

Comment:
I was wondering this too this morning (but have been out all day -- still vacationing). As I recall we now have a multinomial routine elsewhere, ie in Rcpp's `rmulitnom` wrapping R's own.

But we can keep these cleanups for another time.